heroku/github-triage
# created index route

## Why the change?

We need an index route!

## What’s the plan?


## What does it look like?

<img width="545" alt="Screen Shot 2019-06-07 at 3 20 45 PM" src="https://user-images.githubusercontent.com/6692349/59136385-c30b1200-8937-11e9-85ba-6809009d47f7.png">

## How do I verify the change?



1. Visit the review app
2. Verify that the index route looks like the screenshot above
